add implementation scalability as an explicit goal (#157)
ietf-rats-wg/draft-ietf-rats-msg-wrap: draft-ietf-rats-msg-wrap.md

but can instead utilize the CMW format to be agnostic to the attestation
technology.

> A further goal is to ensure the scalability of implementations.

Reviewer: ```suggestion
A further design goal is to extensibility.
```

> This means that adding new message handlers should not change the core of the processor, and that a CMW stack can be designed to offer a plug-in interface for both encoding and decoding.

Reviewer: ```suggestion
This means that adding support for new conceptual messages and new attestation technologies should not change the core of the processor, and that a CMW stack can be designed to offer a plug-in interface for both encoding and decoding.
```

> To achieve this, the format must provide consistent message encapsulation and strong, explicit typing.

Reviewer: ```suggestion
To achieve this, the format must provide consistent message encapsulation and explicit typing.
```
